common-rendering/src/components/keyEvents.tsx

```diff
@@ -164,23 +175,23 @@ const timeStyles = (supportsDarkMode: boolean): SerializedStyles => css`
 const ListItem = ({ keyEvent, format, supportsDarkMode }: ListItemProps) => {
 	return (
 		<li css={listItemStyles(supportsDarkMode)}>
-			<div css={timeTextWrapperStyles}>
-				<time
-					dateTime={keyEvent.date.toISOString()}
-					data-relativeformat="med"
-					title={keyEvent.date.toLocaleTimeString()}
-					css={timeStyles(supportsDarkMode)}
-				>
-					{timeAgo(keyEvent.date.getTime())}
-				</time>
-				<Link
-					priority="secondary"
-					css={textStyles(format, supportsDarkMode)}
-					href={keyEvent.url}
-				>
-					{keyEvent.text}
-				</Link>
-			</div>
+			<Link
+				priority="secondary"
+				css={linkStyles(supportsDarkMode)}
+				href={keyEvent.url}
+			>
+				<div css={timeTextWrapperStyles}>
+					<time
+						dateTime={keyEvent.date.toISOString()}
+						data-relativeformat="med"
+						title={keyEvent.date.toLocaleTimeString()}
```

Comment:
In [this PR](https://github.com/guardian/dotcom-rendering/pull/4254) I got a bit fancy with the title, maybe we can bring something similar over to here too?